CODE_OF_CONDUCT.md

```diff
@@ -1,5 +1,17 @@
 # Contributor Covenant Code of Conduct
 
+## Documentation index
+Please familiarise yourself with our:
+- [Code of conduct](https://github.com/bbc/psammead/blob/latest/CODE_OF_CONDUCT.md) (you are here)
+- [Code Standards and Ways of Working](https://github.com/bbc/simorgh/blob/latest/Code-Standards-and-Ways-of-Workinnng.md)
```

Comment:
Typo. `https://github.com/bbc/simorgh/blob/latest/Code-Standards-and-Ways-of-Working.md`

Reply:
@mykeels done